### DISCUSSION TITLE
How to prevent model to be updated to gpt-4 when updating assistant?

### DISCUSSION THREAD

**Discussion by tongshengng** [VRSEN/agency-swarm#93] [Q&A]
Hi all, how can i prevent the model to be updated to `gpt-4` when updating assistant? 

I have specified the model to `gpt-3.5-turbo` in the `settings.json` and updated openai assistants model as `gpt-3.5-turbo`

But every time i run `agency.py` it always change back to `gpt-4`? i just want to learn how about the tool but `gpt-4` is quite expensive

**Comment by nicko-ai** [comment]
You need to specify the model when creating your agent. By default, gpt-4o is used at the time of the writing (April 2025).
